src/app/qgsstatusbarcoordinateswidget.cpp

```diff
@@ -317,7 +317,7 @@ void QgsStatusBarCoordinatesWidget::ensureCoordinatesVisible()
 {
 
   //ensure the label is big (and small) enough
-  int width = std::max( mLineEdit->fontMetrics().width( mLineEdit->text() ) + 10, mMinimumWidth );
+  int width = std::max( mLineEdit->fontMetrics().width( mLineEdit->text() ) + 16, mMinimumWidth );
```

Comment:
this really should be something like

    + mLineEdit->fontMetrics().width( 'X') * 0.2 

(tweaking the 0.2 factor to get the desired margin). Otherwise it's not hi-dpi friendly.

Reply:
I guess QLineEdit margins don't depend on the font metrics. I was changing font size and it was always 6 pixels missing for Breeze and Oxygen (while older styles always fit perfectly, like in Windows).

Reply:
> I guess QLineEdit margins don't depend on the font metrics. 

They should though -- 16 pixels on a hidpi screen is basically 0, so a use on breeze/oxygen on a hidpi screen will probably also see this issue. @elpaso can you confirm?